src/main/java/mate/controller/GetAllDriversController.java

```diff
@@ -0,0 +1,25 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.DriverService;
+
+public class GetAllDriversController extends HttpServlet {
+    private final Injector injector = Injector.getInstance("mate");
+    private final DriverService driverService = (DriverService) injector
+            .getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        List<Driver> allDrivers = driverService.getAll();
+        req.setAttribute("drivers", allDrivers);
```

Comment:
Let's make the naming of lists more consistent, you already called the list of cars in this way
```suggestion
        req.setAttribute("drivers", driverList);
```